File: scripts/8_delete_w2.js
const weaviate = require('weaviate-client');

const wClassNames = ["Book", "Author"];

var client = weaviate.client({
    scheme: "http",
    host: "localhost:8090"
});

for (wClassName of wClassNames) {
    client.schema
        .classDeleter()
        .withClassName(wClassName)
        .do()
        .then(res => {
            console.log(res);
        })
        .catch(err => {
            console.error(err)
        });
}

File: scripts/8_delete_w2.go
package main

import (
	"context"
	"fmt"

	"github.com/semi-technologies/weaviate-go-client/v4/weaviate"
)

func main() {
	hostW2 := "localhost:8090"

	ctx := context.Background()
	client := weaviate.New(weaviate.Config{Scheme: "http", Host: hostW2})

	fmt.Println("\nDeleting all schemas")
	err := client.Schema().AllDeleter().Do(ctx)
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Println("Deleted")
}
